### PR TITLE
Address issue 169 air density, M third body, catch unexpected JSON

### PR DESCRIPTION
**Pull request by carl-drews**
The changes to music_box.py are similar to what Kyle has checked into the main music-branch branch already. The other changed files catch some key errors that the python code was not expecting to see in the JSON configurations.

### CODE REVIEW

**Review by K20shores — DISMISSED** [review]

**Review by boulderdaze — DISMISSED** [review]
Looks good!

File: src/acom_music_box/music_box_reaction_list.py [hunk 1]
```diff
@@ -114,12 +117,14 @@ def get_reactants_from_JSON(self, reaction, species_list):
...
 
-        for reactant, reactant_info in reaction['reactants'].items():
-            match = filter(lambda x: x.name == reactant, species_list.species)
-            species = next(match, None)
-            quantity = reactant_info['qty'] if 'qty' in reactant_info else None
+        if ('reactants' in reaction.keys()):
+            for reactant, reactant_info in reaction['reactants'].items():
+                match = filter(lambda x: x.name == reactant, species_list.species)
+                species = next(match, None)
+                #logger.info("reactant = {}   species = {}".format(reactant, species))
```
Comment (mattldawson) [inline src/acom_music_box/music_box_reaction_list.py:124]: please remove commented out code

**Review by mattldawson — APPROVED** [review]
looks great! just one very minor suggestion

File: src/acom_music_box/music_box_reaction_list.py (hunk 1 above)
Comment (carl-drews) [reply src/acom_music_box/music_box_reaction_list.py]: Missed that one! :-(

**Review by mattldawson — DISMISSED** [review]
nice!

**Review by boulderdaze — APPROVED** [review]

**Review by K20shores — APPROVED** [review]

**Comment by carl-drews** [comment]
Not sure why Build not completed:
build Expected — Waiting for status to be reported

**Pull request merged**